### DISCUSSION TITLE
MultiMasters not working with pdnsutil?

### DISCUSSION THREAD

**Discussion by bradley-baker** [PowerDNS/pdns#13366] [Q&A] [answered]
My understanding is that PowerDNS supports multiple masters/primaries. One of our primaries is available over two public IPs ( WAN1/WAN2). So same server just different ways to access it. 

As such we'd like to have our secondaries check both primary IP addresses. I tried using the following command as from a few other github posts it seems like the primaries should be separated by commas:

pdnsutil change-secondary-zone-primary dailyxtratravel.co.nz 66.46.118.138,158.106.84.10

But I get this error:
Error: Unable to convert presentation address '66.46.118.138,158.106.84.10'

Is there some way to set two primaries for a secondary zone?

**Comment by aj-gh** [comment, marked as answer]
The [documentation](https://doc.powerdns.com/authoritative/manpages/pdnsutil.1.html) suggest a syntax of `pdnsutil change-secondary-zone-primary ZONE PRIMARY [PRIMARY]…`, further stating "All PRIMARYs need to to be space-separated IP addresses with an optional port".
Did you try that?

**Comment by bradley-baker** [comment]
I saw somewhere else that it should be comma separated. Sorry - space separated does work.
